### PR TITLE
Disable Renovate dependency dashboard

### PR DESCRIPTION
**Pull request by ccremer**
* Disables Renovate Dependency dashboard for commodore and component template
* Follows https://github.com/projectsyn/modulesync-control/pull/51

(Value of this Dashboard isn't that great)

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues.

### CODE REVIEW

File: commodore/component-template/{{ cookiecutter.slug }}/renovate.json [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
   "extends": [
-    "config:base"
+    "config:base",
+    ":gitSignOff",
```
Comment (simu) [inline commodore/component-template/{{ cookiecutter.slug }}/renovate.json:4]: I don't think we need Git signed-off-by for components.

 ```suggestion
```

File: renovate.json [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {
   "extends": [
-    "config:base"
+    "config:base",
+    ":gitSignOff",
```
Comment (simu) [inline renovate.json:4]: same here
```suggestion
```

**Review by simu — CHANGES_REQUESTED** [review]

File: commodore/component-template/{{ cookiecutter.slug }}/renovate.json (hunk 1 above)
Comment (ccremer) [reply commodore/component-template/{{ cookiecutter.slug }}/renovate.json:4]: Right, but I see no harm either.
Do you prefer it without?

Comment (simu) [reply commodore/component-template/{{ cookiecutter.slug }}/renovate.json:4]: I don't really care either way

Comment (ccremer) [reply commodore/component-template/{{ cookiecutter.slug }}/renovate.json:4]: Ok :) Would you mind approving the PR? Merging is blocked with "change requested"

**Review by simu — APPROVED** [review]

**Pull request merged**